core/arch/arm/kernel/generic_core_bootcfg.c

```diff
@@ -149,6 +146,9 @@ static struct map_area bootcfg_memory_map[] = {
 	{
 	 .type = DEVICE1_TYPE,
 	 .pa = DEVICE1_BASE, .size = DEVICE1_SIZE,
+#ifdef DEVICE1_VA_BASE
```

Comment:
@pascal-brand-st-dev @jenswi-linaro 
Some devices' PA base address conflicts with TA. If VA=PA is a must, there is no way to access these devices in optee_os for i.MX6UL.